packages/lib-sourcify/test/validation.spec.ts

```diff
@@ -1,267 +0,0 @@
-        invalidPath,
-      ];
-      const checkedContracts = await checkPaths(
-        solc,
-        vyperCompiler,
-        paths,
-        ignoring,
-      );
-
-      chai.expect(ignoring).to.deep.equal([invalidPath]);
-      expectationsOfSingle(checkedContracts);
-    });
-
-    async function checkSingleWithModifiedEnding(
-      directoryName: string,
-      expectedLineEnd: string,
-      expectedFileEnd: string,
-    ) {
-      const ignoring: string[] = [];
-      const directory = path.join(
-        __dirname,
-        'validation',
-        'files',
-        directoryName,
-      );
-
-      const filePath = path.join(directory, '1_Storage.sol');
-      const content = fs.readFileSync(filePath).toString();
-
-      const nCount = (content.match(/\n/g) || []).length;
-      const rnCount = (content.match(/\r\n/g) || []).length;
-      if (expectedLineEnd === '\n') {
-        chai.expect(rnCount).to.equal(0);
-      } else {
-        chai.expect(nCount).to.equal(rnCount);
-      }
-
-      const endLength = expectedFileEnd.length;
-      const fileEnd = content.slice(content.length - endLength);
-      chai.expect(fileEnd).to.equal(expectedFileEnd);
-
-      const checkedContracts = await checkPaths(
-        solc,
-        vyperCompiler,
-        [directory],
-        ignoring,
-      );
-
-      chai.expect(ignoring).to.be.empty;
-      chai.expect(checkedContracts).to.have.a.lengthOf(1);
-
-      const contract = checkedContracts[0];
-      chai.expect(contract.name).to.equal('Storage');
-      chai.expect(contract.isValid()).to.be.true;
-    }
-
-    it('should replace \\r\\n with \\n', function () {
-      checkSingleWithModifiedEnding('single-replace-with-n', '\r\n', '}');
-    });
-
-    it('should replace \\n with \\r\\n', function () {
-      checkSingleWithModifiedEnding('single-replace-with-rn', '\n', '}');
-    });
-
-    it('should add a trailing \\r\\n', function () {
-      checkSingleWithModifiedEnding('single-add-trailing-rn', '\r\n', '}');
-    });
-
-    it('should add a trailing \\n', function () {
-      checkSingleWithModifiedEnding('single-add-trailing-n', '\n', '}');
-    });
-
-    it('should remove a trailing \\r\\n', function () {
-      checkSingleWithModifiedEnding(
-        'single-remove-trailing-rn',
-        '\r\n',
-        '\r\n',
-      );
-    });
-
-    it('should remove a trailing \\n', function () {
-      checkSingleWithModifiedEnding('single-remove-trailing-n', '\n', '\n');
-    });
-
-    it('should validate a file with two trailing n', function () {
-      // this fails if not checking the original file
-      checkSingleWithModifiedEnding('single-keep-original', '\n', '\n\n');
-    });
-  });
-});
-
-describe('Unit tests', function () {
-  const pathContent = {
-    path: './validation/files/hardhat-output/output.json',
-    content: JSON.stringify(hardhatOutput),
-  };
-  it('Should extractHardhatMetadataAndSources', async function () {
```

Comment:
This test was completely removed. I think we should not remvoe test cases for functions which we still have in the same form as before.

Reply:
Right! I completely lost that function since it's not used directly.